odoo/addons/base/models/ir_model.py

```diff
@@ -523,9 +523,9 @@ class IrModelFields(models.Model):
     copied = fields.Boolean(string='Copied',
                             compute='_compute_copied', store=True, readonly=False,
                             help="Whether the value is copied when duplicating a record.")
-    related = fields.Char(string='Related Field', help="The corresponding related field, if any. This must be a dot-separated list of field names.")
+    related = fields.Char(string='Related Field Name', help="The corresponding related field, if any. This must be a dot-separated list of field names.")
     related_field_id = fields.Many2one('ir.model.fields', compute='_compute_related_field_id',
-                                       store=True, string="Related field", ondelete='cascade')
+                                       store=True, string="Related Field", ondelete='cascade')
```

Comment:
IMO, it is still confusing. It is actually only the first field in the related sequence.

Reply:
Do you think `Related Field Record` is better ? We don't usually add record adjective for m2o but maybe here we should

Reply:
> IMO, it is still confusing. It is actually only the first field in the related sequence.

It's actually the _last_ field in the sequence, i.e., the field that gives the value to the related field.

By the way, the sole purpose of `related_field_id` is its foreign key: if you delete the target field, the current field is automatically deleted.  It does not appear in views (except maybe if you look for something to search on).

Reply:
> except maybe if you look for something to search on

That's how I found this I was trying to do a search on the "Related Field" displayed in the form view and got confused when it appeared 2 times in the search list.